notebooks/ch12/LR_Case_Study.md

```diff
@@ -0,0 +1,792 @@
+
+Since we can easily see that this lates births value is less than 10, we can just filter our dataset by selecting only rows where late births is over 10.
+
+
+```python
+drinking_data = drinking_data[drinking_data['Late births'] >= 10]
+drinking_data
+```
+
+
+
+
+<div>
+<style>
+    .dataframe thead tr:only-child th {
+        text-align: right;
+    }
+
+    .dataframe thead th {
+        text-align: left;
+    }
+
+    .dataframe tbody tr th {
+        vertical-align: top;
+    }
+</style>
+<table border="1" class="dataframe">
+  <thead>
+    <tr style="text-align: right;">
+      <th></th>
+      <th>Bias</th>
+      <th>Urban population</th>
+      <th>Late births</th>
+      <th>Wine</th>
+      <th>Liquor</th>
+      <th>Cirrhosis</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>0</th>
+      <td>1</td>
+      <td>44.0</td>
+      <td>33.2</td>
+      <td>5</td>
+      <td>30</td>
+      <td>41.2</td>
+    </tr>
+    <tr>
+      <th>1</th>
+      <td>1</td>
+      <td>43.0</td>
+      <td>33.8</td>
+      <td>4</td>
+      <td>41</td>
+      <td>31.7</td>
+    </tr>
+    <tr>
+      <th>2</th>
+      <td>1</td>
+      <td>48.0</td>
+      <td>40.6</td>
+      <td>3</td>
+      <td>38</td>
+      <td>39.4</td>
+    </tr>
+    <tr>
+      <th>...</th>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+    </tr>
+    <tr>
+      <th>43</th>
+      <td>1</td>
+      <td>44.0</td>
+      <td>35.3</td>
+      <td>3</td>
+      <td>32</td>
+      <td>51.6</td>
+    </tr>
+    <tr>
+      <th>44</th>
+      <td>1</td>
+      <td>63.0</td>
+      <td>39.3</td>
+      <td>8</td>
+      <td>40</td>
+      <td>55.7</td>
+    </tr>
+    <tr>
+      <th>45</th>
+      <td>1</td>
+      <td>58.0</td>
+      <td>43.8</td>
+      <td>13</td>
+      <td>57</td>
+      <td>55.5</td>
+    </tr>
+  </tbody>
+</table>
+<p>44 rows × 6 columns</p>
+</div>
+
+
+
+How does our scatterplot look now without that outlier?
+
+
+```python
+# HIDDEN
+plt.scatter(drinking_data['Late births'], drinking_data['Cirrhosis'])
+plt.title("Late births vs. Cirrhosis")
+plt.xlabel("Late births")
+plt.ylabel("Cirrhosis");
+```
+
+
+![png](LR_Case_Study_files/LR_Case_Study_31_0.png)
+
+
+Much better!
+
+
+```python
+# HIDDEN
+plt.scatter(drinking_data['Wine'], drinking_data['Cirrhosis'])
+plt.title("Wine vs. Cirrhosis")
+plt.xlabel("Wine")
+plt.ylabel("Cirrhosis");
+```
+
+
+![png](LR_Case_Study_files/LR_Case_Study_33_0.png)
+
+
+This looks like a pretty strong linear relationship between wine and cirrhosis.
+
+
+```python
+# HIDDEN
+plt.scatter(drinking_data['Liquor'], drinking_data['Cirrhosis'])
+plt.title("Liquor vs. Cirrhosis")
+plt.xlabel("Liquor")
+plt.ylabel("Cirrhosis");
+```
+
+
+![png](LR_Case_Study_files/LR_Case_Study_35_0.png)
+
+
+There seems to be a somewhat linear relationship between liquor and cirrhosis, though it is definitely not as strong as that between wine and cirrhosis.
```

Comment:
don't think the qualification is necessary, doesn't add instructional value and the plot is just right above

Reply:
hmm i think this line is just about making observations about our data and not necessarily relevant to the linear regression